diff --git a/node_modules/@strapi/admin/dist/admin/admin/src/components/Context.js b/node_modules/@strapi/admin/dist/admin/admin/src/components/Context.js
index bdecff1..a5a3627 100644
--- a/node_modules/@strapi/admin/dist/admin/admin/src/components/Context.js
+++ b/node_modules/@strapi/admin/dist/admin/admin/src/components/Context.js
@@ -41,7 +41,7 @@ var ContextSelector__namespace = /*#__PURE__*/_interopNamespaceDefault(ContextSe
             children: children
         });
     };
-    function useContext(consumerName, selector, shouldThrowOnMissingContext) {
+    function useContext(consumerName, selector, shouldThrowOnMissingContext = true) {
         return ContextSelector__namespace.useContextSelector(Context, (ctx)=>{
             // The context is available, return the selected value
             if (ctx) return selector(ctx);
diff --git a/node_modules/@strapi/admin/dist/admin/admin/src/components/Context.mjs b/node_modules/@strapi/admin/dist/admin/admin/src/components/Context.mjs
index 0004624..5887d9f 100644
--- a/node_modules/@strapi/admin/dist/admin/admin/src/components/Context.mjs
+++ b/node_modules/@strapi/admin/dist/admin/admin/src/components/Context.mjs
@@ -19,7 +19,7 @@ import * as ContextSelector from 'use-context-selector';
             children: children
         });
     };
-    function useContext(consumerName, selector, shouldThrowOnMissingContext) {
+    function useContext(consumerName, selector, shouldThrowOnMissingContext = true) {
         return ContextSelector.useContextSelector(Context, (ctx)=>{
             // The context is available, return the selected value
             if (ctx) return selector(ctx);
